.travis.yml

```diff
@@ -23,13 +23,15 @@ env:
         - PATH="$TRAVIS_BUILD_DIR/bin:$PATH"
 
     matrix:
+        - SINGLE=true
         - DIST=       DMD=dmd-transitional
         - DIST=trusty DMD=dmd1
         - DIST=xenial DMD=2.070.2.s12
 
 install:
-    - beaver dlang install
+    - test -n "$SINGLE" || beaver dlang install
 
 script:
-    - beaver docker run test/hello
-    - beaver dlang make
+    - test -n "$SINGLE" || beaver docker run test/hello
+    - test -n "$SINGLE" || beaver dlang make
+    - if test -n "$SINGLE"; then test/single-tests; fi
```

Comment:
I'm used to not use `&&` because if `sh -e` is set, it fill stop and make the script fail just when the first test fails. But in this case it should be the same because is the last command in the script. If the `test` fails, then travis will see the whole script failing and will mark the build as failed.